ReactiveCocoa/UIKit/UIViewController.swift

```diff
@@ -35,23 +35,9 @@ extension Reactivity where Reactant: UIViewController {
 		return reactant.trigger(for: #selector(UIViewController.viewWillAppear(_:)))
 	}
 
-	public typealias DismissingCompletion = ((Void) -> Void)?
-	public typealias DismissingInformation = (animated: Bool, completion: DismissingCompletion)
-
-	/// Wraps a viewController's `dismissViewControllerAnimated` function in a bindable property.
-	/// It mimics the same input as `dismissViewControllerAnimated`: a `Bool` flag for the animation
-	/// and a `(Void -> Void)?` closure for `completion`.
-	/// E.g:
-	/// ```
-	/// //Dismissed with animation (`true`) and `nil` completion
-	/// viewController.dismissAnimated <~ aProducer.map { _ in (true, nil) }
-	/// ```
-	/// The dismissal observation can be made either with binding (example above)
-	/// or `viewController.dismissViewControllerAnimated(true, completion: nil)`
-	public var dismissAnimated: BindingTarget<DismissingInformation> {
```

Comment:
While `dismissAnimated` the fringe case feels appropriate as an `Action`, it could open a confusing gap.

```
vc.rac.dismissAnimated.events.filter { $0 == .completed }
vc.rac.trigger(for: #selector(dismiss(animated:)))
```

They look the same, and the users would likely expect them to be equivalent. But the first one only works on calls done through the `Action`, while the later is triggered upon all calls.

Therefore, it was axed in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3210/commits/1797e79ae8c4e401ba52a40a13e2a412d0497b76, and replaced with just a `dismissedPresented` trigger.